x-team/xp-ui
# Switch RichTextEditor with InputField textarea on Notes

**Release Type:** *Breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3824

## Description

Switch RichTextEditor with InputField textarea on Notes

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3824-textbox-for-notes-not-displayed` | [XP](https://github.com/x-team/xp/pull/2105)

## Steps to Test or Reproduce

1. Go to http://localhost:9001/?path=/story/ui-components-notesfeed--basic-usage
1. Try editing the note, observe its no longer a RichTextEditor but the InputField in textarea mode

![2020-07-29 09 25 13](https://user-images.githubusercontent.com/131859/88799917-a1916900-d17d-11ea-890b-80253c3c7994.gif)
